lib/improver/utilities/cube_metadata.py

```diff
@@ -95,6 +95,43 @@ def update_stage_v110_metadata(cube):
     return True
 
 
+def update_units(item, changes):
+    """Update the units on the item supplied. Options are to either
+    'set' or 'convert' the units
+
+    Args:
+        item (iris.cube.Cube, iris.coords.DimCoord or iris.coords.AuxCoord):
+            Item that will have its units modified.
+        changes (dict):
+            Dictionary of the formats shown below for specifying whether the
+            units will be set or converted::
+            "units": {
+                "action": "set",
```

Comment:
This seems a bit dangerous.  Can you think of a use case where we'd need to set units without taking account of what already exists?

Reply:
The `action` keyword has been removed.